api_spec.yaml

```diff
@@ -6851,3 +6841,18 @@ components:
     oauth2RedirectUri:
       type: string
       example: 'https://example.com/auth/callback'
+      
+    oauth2ClientInfo:
+      type: object
+      properties: 
+        clientId:
+          type: string
+          example: "TU/52WOcktSv99zqaAZuWJG9BSoS0aRLfCbep8rFEwk="
```

Comment:
I think we will be using v4 UUIDs for this. So give an example with a v4 UUID

Reply:
Added v4 uuid example.